index.js

```diff
@@ -132,12 +132,14 @@ function runNightwatch (done) {
 				 */
 				function (cb) {
 					Nightwatch.runner(argv, function (status) {
+						var err = null;
 						if (status) {
 							console.log([moment().format('HH:mm:ss:SSS')] + ' kne: tests passed');
 						} else {
 							console.log([moment().format('HH:mm:ss:SSS')] + ' kne: tests failed');
+							err = new Error('kne: nightwatch runner return error status code: 0');
```

Comment:
isn't status code: 0 semantically a legal return in unix? should this be status code -1 or 1 or something that is usually considered a return error?